src/Nethermind/Nethermind.Network.Discovery.Test/NettyDiscoveryHandlerTests.cs

```diff
@@ -171,6 +174,33 @@ public async Task NeighborsSentReceivedTest()
             AssertMetrics(210);
         }
 
+        [Test]
+        [TestCase("010203")]
```

Comment:
Can be just TestCase instead of two attrs. Could you confirm it handles empty, and big packets well, please?

Reply:
Regarding other cases. This test should actually just test that v4 handler forwards messages to v5 handler if one doesn't match the v4 format. It contained a placeholder code I forgot to replace. Fixed it, but I don't see a need for this test specifically to cover different messages sizes, as internal forwarding logic doesn't depend on it.

Reply:
Seemed to me it might reveal potential issues on receiver(v4) side

Reply:
Added validation on packet sizes for both handlers